printc/__init__.py

```diff
@@ -1,17 +1,21 @@
 from sys import stdout
 
+    
+class Color:
+    def __init__(self, c=15, flush=False):
+        self.c = c
+        self.flush = flush
 
-def add_color_esc_codes(text: str, c: int) -> str:
-    """Surround string with escape color code and
-    """
-    cnum = 16
-    base = 30 if c < cnum / 2 else 90
-    color_code = f'\033[0;{base + c % 8};40m'
-    reset_code = '\033[0m'
-    return f'{color_code}{text}{reset_code}'
+    def __enter__(self):
```

Comment:
Imho thats an overkill. I would go with fstring, but maybe something like  f'{color_code}{text}{reset_code}', where text could be something similar to what `print` does - sep.join(args)